code/game/objects/items/weapons/clown_items.dm

```diff
@@ -238,43 +240,80 @@
 
 	if(isitem(target))
 		var/obj/item/target_item = target
+		if(target_item.current_glue_state != GLUE_STATE_NONE) //Check to see if its glued first.
```

Comment:
not that I guess this is a big deal, but it means you can't superglue something currently normal-glued

Reply:
I was going to allow upgrading from normal glue to super, guess I'll do that.